src/shared/database/repositories/users.ts

```diff
@@ -48,11 +57,22 @@ export const usersRepository = () => {
     return db.update(users).set({ deletedAt: new Date() }).where(eq(users.id, id)).returning();
```

Comment:
```suggestion
    return await db.update(users).set({ deletedAt: new Date() }).where(eq(users.id, id)).returning();
```

we need to await here, otherwise the stack trace is lost